doc/src/tutorials/creating-packages.md

```diff
@@ -605,9 +605,20 @@ may fit your package better.
 
      * `CPLEX.jl` wraps the `CPLEX` library, which can be identified easily in a web search.
      * `MATLAB.jl` provides an interface to call the MATLAB engine from within Julia.
+
 7. Avoid naming a package closely to an existing package
      * `Websocket` is too close to `WebSockets` and can be confusing to users. Rather use a new name such as `SimpleWebsockets`.
 
+8. Avoid using a distinctive name that is already in use in another unrelated project.
+     * Don't use the names `Tkinter.jl`, `TkinterGUI.jl`, etc. for a package that is unrelated
+       to the popular `tkinter` python package. Users are likely to assume an affiliation that
```

Comment:
> Users are likely to assume an affiliation that does not exist

But this also applies to the case below:

> It's okay to name a package `AlphaZero.jl` ... even without explicit affiliation

so here users would also assume an affiliation that doesn't exist, or?

Reply:
```suggestion
       to the popular `tkinter` python package. Users are likely to assume a connection that
```